### ISSUE TITLE
what about a styled panel?

### ISSUE THREAD

**Issue by pjrobin** [EtheaDev/StyledComponents#38]
I'm missing a style panel in your lib, something like this https://stackoverflow.com/questions/5755719/rounded-and-titled-tpanel-in-delphi-7

**Comment by carloBarazzetta** [comment]
yes, because my library is focused on "buttons"... and "dialogs"... maybe in the future.

**Comment by edalab-xrg** [comment]
I also need a rounded corner panel。Please arrange it into your plan。thanks

**Comment by carloBarazzetta** [comment]
As soon as possible...

**Comment by carloBarazzetta** [comment]
Sorry but It's not yet implemented...
